pkg/api/postage.go

```diff
@@ -10,11 +10,12 @@ import (
 	"strconv"
 
 	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
 	"github.com/gorilla/mux"
 )
 
 type postageCreateResponse struct {
-	BatchID []byte `json:"batchID"`
+	BatchID swarm.Address `json:"batchID"`
```

Comment:
not sure we'd actually want to use this type here, strictly speaking, this isn't an `address`

Reply:
i only used this because it has the right encoding. `[]byte` uses base64 unfortunately.

Reply:
`[]byte` and implement `json.Marshaler`?

Reply:
you cannot implement anything on `[]byte`. I made a custom type for this field.

Reply:
i meant the define the JSON marshaler on `postageCreateResponse`